fixed password reset

edwardmccaughan opened the pull request.

jottr commented on app/views/devise/passwords/edit.haml:
```diff
@@ -0,0 +1,22 @@
+        = f.hidden_field :reset_password_token
+        
+        .form-group
+          = f.label :password, "Neue passwort", :class => "control-label col-sm-2"
```
Please fix spelling. It should be "Neues Passwort."

jottr commented on app/views/devise/registrations/new.html.erb:
```diff
@@ -34,7 +34,7 @@
 
         <div class="form-group">
-          <%= f.label :password_confirmation, :class => "col-sm-2 control-label" %>
+          <%= f.label :password_confirmation, "Passwort bestätigung", :class => "col-sm-2 control-label" %>
```
This should be "Passwortbestätigung"

edwardmccaughan replied:
When it's all one word, it's too big to fit in the label space, do you want me to change the label column to col-sm-3 (and make the input fields smaller) ?

jottr replied:
Yes please. Could you provide a screenshot of the design change? Thx

xhr15 replied:
Do whatever is needed to make "Passwortbestätigung" fit :)

edwardmccaughan commented:
ok, I fixed those comments and I changed the register page slightly to have more space for the input labels:
![image](https://cloud.githubusercontent.com/assets/141722/15823092/7c52f1e6-2bf9-11e6-935c-be3506fd80f2.png)

jottr commented:
Is there a way you can fix the alignment so it has the same spacing for all forms?
Also, please use the word "Passwort" not "Password". 
Thx.

edwardmccaughan commented:
@jottr like this?:
![image](https://cloud.githubusercontent.com/assets/141722/15823760/5da36ee4-2bfc-11e6-9315-02a43c7ca554.png)

jottr commented:
Yes, perfect.
